soco/core.py

```diff
@@ -1251,6 +1263,96 @@ def dialog_mode(self, dialog_mode):
             ]
         )
 
+    @property
+    def surround_ambient_enabled(self):
+        """Return True if surround ambient mode is enabled for surround music
+        playback.
+
+        If False, playback on surround speakers uses full volume.
+
+        Note: does not apply to TV playback.
+        """
+        if not self.is_soundbar:
+            return None
+
+        response = self.renderingControl.GetEQ(
+            [("InstanceID", 0), ("EQType", "SurroundMode")]
+        )
+        return not int(response["CurrentValue"])
+
+    @surround_ambient_enabled.setter
+    @only_on_soundbars
+    def surround_ambient_enabled(self, value):
+        """Toggle surround music playback mode. True = ambient mode.
```

Comment:
I just realized that having the ambient mode as True makes using the eventing API awkward (ambient mode is report as `0`), so this may need to be changed to be other way around. IIRC I didn't do it previously as I couldn't think of a good name for the "full mode".